renovate.json

```diff
@@ -1,6 +1,6 @@
 {
-  "enabled": false,
+  "enabled": true,
   "extends": ["@artsy:lib"],
-  "assignees": ["zephraph"],
+  "assignees": ["eessex"],
   "reviewers": []
 }
```

Comment:
What do you think about running renovate less frequently so that it doesn't take up too much Circle CI capacity? I made this change yesterday as a temporary patch after we noticed a significant amount of Circle CI build queueing.

Looks like Volt has something similar configured which we could pull from: https://github.com/artsy/volt/blob/master/renovate.json#L2-L5

Here's a elated Slack thread from yesterday: https://artsy.slack.com/archives/C03J4L2KK/p1595448447030800